src/test/rgw/CMakeLists.txt

```diff
@@ -222,8 +222,4 @@ if(WITH_RADOSGW_AWS_ENDPOINT)
   add_executable(unittest_rgw_aws test_rgw_aws.cc)
   target_include_directories(unittest_rgw_aws PRIVATE "${CMAKE_SOURCE_DIR}/src/aws-sdk-cpp/aws-cpp-sdk-core/include")
   target_link_libraries(unittest_rgw_aws ${rgw_libs} ${UNITTEST_LIBS})
-  install(TARGETS
```

Comment:
don't we need this line to install the unit test binary?

Reply:
we don't install unit tests. they are built and placed into build/bin so we can run them in the test environment, but they are not needed on the machine where we install ceph